src/main/java/com/purbon/kafka/topology/TopologyObjectBuilder.java

```diff
@@ -55,10 +56,11 @@ private static PlanMap buildPlans(String plansFile) throws IOException {
     if (isDir) {
       Files.list(Paths.get(fileOrDir))
           .sorted()
-          .map(path -> parser.deserialise(path.toFile()))
-          .forEach(subTopology -> topologies.add(subTopology));
+          .map(Path::toFile)
+          .map(parser::deserialize)
+          .forEach(topologies::add);
     } else {
-      Topology firstTopology = parser.deserialise(new File(fileOrDir));
+      Topology firstTopology = parser.deserialize(new File(fileOrDir));
```

Comment:
can we centre to use the s here? changing to z is mostly to be American English "compliant". That would appreciate it ;-) ref: https://en.wikipedia.org/wiki/American_and_British_English_spelling_differences#-ise.2C_-ize_.28-isation.2C_-ization.29

Reply:
Fair point. We can add this somewhere in the docs to make this decision explicit.

Most libraries use `-ize` though, as Jackson that its use in the project. I suggested the change to keep those consistent. It's a bit confusing to have `*Deserializer` classes and `deserialise` methods.